project-X.rb

```diff
@@ -0,0 +1,65 @@
+require 'fileutils'  # This library helps to remove STOP from the end of the list
+
+#Create The New List Part#
+puts "Write the name of your new list:" #
+name_of_file = gets.chomp.to_s.<< ".txt"
+puts "#{name_of_file} list was succesfully created!\n Remember! If you want to stop writing words, just write 'STOP'."
+file = File.open(name_of_file, "w+")
+eng_error_text = "Your word isn't English, please write English word"
+rus_error_text = "Your word isn't Russian, please write Russian word"
+
+#English Word Part#
+
+loop do # This huge loop helps repeat asking user to put English and Russian word again and again till user puts STOP
+  puts "Write English word:"
+  users_input_eng = gets
+
+  eng_correction = users_input_eng.match?(/[qwertyuiopasdfghjklzxcvbnmQWERTYUIOPASDFGHJKLZXCVBNM]/i) # Checking, if users input is correct.
+  # If user write number, symbol or Russian word, program will ask to write English word again and puts the message with error
+
+  while eng_correction == false # This loop repeat asking Eng word if input is uncorrect
+    eng_correction = users_input_eng.match?(/[qwertyuiopasdfghjklzxcvbnmQWERTYUIOPASDFGHJKLZXCVBNM]/i) # Checks again
+    if eng_correction == false
+      puts eng_error_text
+      users_input_eng = gets
+    else
+      break
+    end
+  end
```

Comment:
Let's add some formatting here. I would add an extra line in order to see where is the end of the loop.